Add info on broken iptables versions (#14)
k3s-io/docs: docs/advanced/advanced.md

INFO[2019-01-22T15:16:20.541049100-07:00] Run: k3s kubectl                             
```

The output will likely be much longer as the agent will create a lot of logs. By default, the server
will register itself as a node (run the agent).

## Additional OS Preparations

### Additional preparation for Debian "buster" based distributions

> Several popular OSes based on Debian "buster" ship a version of iptables, between v1.8.0-v1.8.4. These versions contain a bug which cause the accumulation of too many rules which will bog down and crash the K3s server. See [this issue](https://github.com/k3s-io/k3s/issues/3117) for more background. 

Reviewer: ```suggestion
Several popular Linux distributions based on Debian "buster" ship a version of IPTables between v1.8.0-v1.8.4. These versions contain a bug which causes the accumulation of duplicate rules, which negatively affects the performance and stability of the node. See [Issue #3117](https://github.com/k3s-io/k3s/issues/3117) for more background. 
```

Author: It appears that iptables is never capitalized in any of the documentation I could find online. Every other change was added.

Switching from nftables mode to legacy iptables mode will bypass this issue.

```bash
sudo iptables -F
sudo update-alternatives --set iptables /usr/sbin/iptables-legacy
sudo update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
sudo reboot
```

> Alternatively, K3s ships which a working version of ipTables (v1.8.6) which does not cause a crash. You can replace iptables on your system:

Reviewer: ```suggestion
Alternatively, K3s ships which a working version of ipTables (v1.8.6) which functions properly. You can replace iptables on your system:
```

Author: Fixed.

```bash
sudo apt remove iptables nftables -y
sudo reboot
export PATH="/var/lib/rancher/k3s/data/current/bin/:/var/lib/rancher/k3s/data/current/bin/aux:$PATH"
```

> K3s will now use it's packaged version of iptables.

Reviewer: ```suggestion
K3s will now use its packaged version of iptables.
```

Author: Fixed
